proposals/csharp-8.0/ranges.md

```diff
@@ -0,0 +1,58 @@
+# Ranges
+
+* [x] Proposed
```

Comment:
I think this section with checkmarks should only be in the championed issue, not the speclet.